lib/curfews.js

```diff
@@ -45,6 +45,28 @@ const curfews = [
             until: DateTime.fromFormat('Dec 11, 2021', 'DD'),
         },
     },
+    {
+        start: DateTime.fromFormat('Dec 10, 2021, 10:00 PM', 'ff'),
+        end: DateTime.fromFormat('Dec 11, 2021, 5:00 AM', 'ff'),
+        repeats: {
+            freq: 'daily',
+            until: DateTime.fromFormat('Jan 13, 2022', 'DD'),
```

Comment:
Curfew end date is Jan 14. To represent that here the date has to be curfew end date + 1 day, which would be Jan 15 in this case. The reason for this is to include Jan 14 in the repetition.

```suggestion
            until: DateTime.fromFormat('Jan 15, 2022', 'DD'),
```